martin/src/args/root.rs

```diff
@@ -157,9 +157,17 @@ impl Args {
 
 #[cfg(any(feature = "pmtiles", feature = "mbtiles", feature = "cog"))]
 fn is_url(s: &str, extension: &[&str]) -> bool {
-    if s.starts_with("http") {
+    if s.starts_with("http") || s.starts_with("s3") {
```

Comment:
i keep wondering if we even need this optimization - if this function is only used for parsing, we might as well just parse the URL and check the schema.  Also, why do we parse the url here, and also have `parse_url`?  I probably did it a while back, but not sure if this duplicity is needed